.github/workflows/build-and-test.yml

```
@@ -0,0 +1,30 @@
+name: build-and-test
+on: [pull_request]
+jobs:
+  test-re2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
```

You can checkout submodules like this:
```suggestion
      - uses: actions/checkout@v2
        with:
          submodules: true
```

Reference [here](https://github.com/actions/checkout#usage).